index.html

```diff
@@ -0,0 +1,927 @@
+                  src="assets/images/rect.svg"
+                  alt="rectangle"
+                />
+                <h2>Frameworks</h2>
+              </li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+
+          <div class="third">
+            <ul class="objects">
+              <li class="rubric skill">
+                <img
+                  class="circle"
+                  src="assets/images/circle.svg"
+                  alt="circle"
+                />
+                <h2>Skills</h2>
+              </li>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <!--This is footer part-->
+
+      <section class="footer">
+        <div class="introduction">
+          <p>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+        </div>
+
+        <div class="form">
+          <form action="https://formspree.io/f/mqknkkgn" method="POST">
+            <ul>
+              <li class="my-name">
+                <label for="my-name"></label>
+                <input
+                  class="text"
+                  type="text"
+                  id="my-name"
+                  name="full_name"
+                  placeholder="Full name"
+                  minlength="3"
+                  maxlength="30"
+                  required
+                />
+                <img
+                  class="break"
+                  src="assets/images/Container.svg"
+                  alt="line"
+                />
+              </li>
+
+              <li class="my-mail">
+                <label for="my-mail"></label>
+                <input
+                  class="text"
+                  type="email"
+                  id="my-mail"
+                  name="user_email"
+                  placeholder="Email address"
+                  required
+                />
+                <img
+                  class="break"
+                  src="assets/images/Container.svg"
+                  alt="line"
+                />
+              </li>
+
+              <li class="blog">
+                <label for="my-msg"></label>
+                <textarea
+                  class="area"
+                  id="my-msg"
+                  name="user_message"
+                  rows="10"
+                  cols="50"
+                  placeholder="Enter text here"
+                  minlength="7"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+
+              <li class="btn">
+                <button class="touch" type="submit">Get in touch</button>
```

Comment:
**Kindly follow the requested style on your design guidelines on the next examples:**
- Implement the `hover` effect at the same color.
<img width="1440" alt="Screen Shot 2021-12-10 at 18 57 02" src="https://user-images.githubusercontent.com/22258274/145658138-77073727-69cc-4eb2-a5f2-5043e1c078c8.png">
#

Improve the design of this section changing `background`, `icons alignment`, etc, as you see in the next examples:

_Figma Template:_

<img width="1440" alt="Screen Shot 2021-12-10 at 18 57 44" src="https://user-images.githubusercontent.com/22258274/145658183-942e4b8a-97d5-41a3-aab2-7b6a1a773958.png">

_Yours:_

<img width="1231" alt="Screen Shot 2021-12-10 at 18 58 18" src="https://user-images.githubusercontent.com/22258274/145658292-e1da0228-932d-426e-a049-7a6c7296f00b.png">

Reply:
I have changed the background for mentioned buttons for hover and pressed states.

The background of the section also is changed.

I changed the width of the icons to fit as closely as possible the desktop version of Figma. But since, in accordance with the requirement of the project in the desktop version, as well as in the mobile version, there should be a full_name, and no separate fields for the first name and for the last, I cannot fully match the desktop version of Figma. 
![footer](https://user-images.githubusercontent.com/52704856/145669189-f9cf4d46-7092-4161-a356-54377c95f024.png)